lib/ansible/utils/collection_loader/_collection_meta.py

```diff
@@ -4,17 +4,13 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-from yaml import load
-try:
-    from yaml import CSafeLoader as SafeLoader
-except ImportError:
-    from yaml import SafeLoader
-
```

Comment:
The changes to this file should also be applied to this one:

https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/_data/legacy_collection_loader/_collection_meta.py